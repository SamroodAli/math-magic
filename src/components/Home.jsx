import { Segment } from 'semantic-ui-react';

const Home = () => (

  <div>
    <Segment vertical>
      <h1>Welcome to Math Magic</h1>
    </Segment>

    <Segment vertical>
      <p>
        Feel free to use our calculator or enjoy reading some Maths quotes.
      </p>
    </Segment>
    <Segment vertical>
      <p>This project is built with React.</p>
    </Segment>
  </div>
);

export default Home;
