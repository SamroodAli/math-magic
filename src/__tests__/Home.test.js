import renderer from 'react-test-renderer';
import Home from '../components/Home';

test('Display panel displays all calculator buttons', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
