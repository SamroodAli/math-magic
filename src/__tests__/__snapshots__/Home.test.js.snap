// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display panel displays all calculator buttons 1`] = `
<h1>
  Welcome to Math Magic
</h1>
`;
